rest_framework_json_api/renderers.py

```diff
@@ -433,6 +427,13 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
 
         serializer = getattr(serializer_data, 'serializer', None)
 
+        # Build a list of included resources
+        include_resources_param = request.query_params.get('include') if request else None
+        if include_resources_param:
+            included_resources = include_resources_param.split(',')
+        else:
+            included_resources = utils.get_default_included_resources_from_serializer(serializer)
```

Comment:
I wanted to make sure the `include` param should override the defaults, which is how you implemented it here.

```
If an endpoint supports the include parameter and a client supplies it, the server MUST NOT include unrequested resource objects in the included section of the compound document.
```

:+1: , looks like this is all good.